camkes/templates/component.common.c

```diff
@@ -84,6 +84,31 @@ int get_instance_affinity(void) {
     return /*? configuration[me.name].get('affinity', options.default_affinity) ?*/;
 }
 
+#ifdef CONFIG_ALLOW_SMC_CALLS
+seL4_CPtr camkes_get_smc_cap(seL4_Word smc_call){
+    seL4_CPtr cap;
+    switch(smc_call) {
+/*# ARM SMC cap allocation #*/
+/*- if 'allow_smc' in configuration[me.name].keys() -*/
+    /*- if configuration[me.name].get('allow_smc') == true -*/
+        /*- if 'allowed_smc_functions' in configuration[me.name].keys() -*/
+            /*- for func_id in configuration[me.name].allowed_smc_functions -*/
```

Comment:
Can we use this to merge the lines?
```suggestion
            /*- for func_id in configuration[me.name].get('allowed_smc_functions', []) -*/
```